### DISCUSSION TITLE
Possibility of a X/Y Axis Label for the most recent value?

### DISCUSSION THREAD

**Discussion by EthanBlank-Git** [Live-Charts/LiveCharts2#546] [Ideas]
First off, thank you for this library, it is by far the best charting library I have used so far, you are amazing!!

Just was wondering if there is any thoughts about an X/Y axis label that would display the current value of the last data point. Similar to how Trading View and other stock market charts have this:
![image](https://user-images.githubusercontent.com/5265408/178045907-6fbac24e-205d-4bd3-96a1-30842c1ffa12.png)

**Comment by beto-rodriguez** [comment]
Hi and thanks!

Yes it will be implemented, but it is not ready yet, we can track this at #490.

**Reply by jasonsullivan** [reply]
Hello @EBTheKidd were you able to implement this based on the enhancement from #490? If so could you share your approach?

Looking for the same capability: #883 

Kind regards
Jason

**Reply by EthanBlank-Git** [reply]
I was able to get it working, with mixed results:
![image](https://user-images.githubusercontent.com/5265408/211846564-cf386cd2-3d36-4d8c-9f7d-710ca6db5fea.png)
It works as described, although my program appeared to crash quite frequently while using it. I believe it is because my chart is constantly updating, although I am not 100% sure. Here is my code (taken from the sample provided in the documentation)
![image](https://user-images.githubusercontent.com/5265408/211847680-42788bbe-7eee-45a9-88a3-57e457202174.png)

Being that it continuously crashes the application (only while trying to use the crosshair), I decided to omit it from my releases until I can track down the issue
